TESTS WITH ERRORS 
-------------------------------
TESTS FAILED  
test04_GM_Hanoi_04 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 122, in test04_GM_Hanoi_04
    self.assertEqual(th.getGameState(), ((3,4,5),(),(1,2)))
AssertionError: Tuples differ: ((2, 3, 4, 5), (), (2,)) != ((3, 4, 5), (), (1, 2))

First differing element 0:
(2, 3, 4, 5)
(3, 4, 5)

- ((2, 3, 4, 5), (), (2,))
?   ---                -

+ ((3, 4, 5), (), (1, 2))
?                  +++


test07_GM_8Puzzle_03 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 175, in test07_GM_8Puzzle_03
    self.assertEqual(p8.getGameState(), ((5,1,4), (6,-1,8), (7,3,2)))
AssertionError: Tuples differ: ((5, 4, -1), (6, 1, 8), (7, 3, 2)) != ((5, 1, 4), (6, -1, 8), (7, 3, 2))

First differing element 0:
(5, 4, -1)
(5, 1, 4)

- ((5, 4, -1), (6, 1, 8), (7, 3, 2))
?       ----

+ ((5, 1, 4), (6, -1, 8), (7, 3, 2))
?      +++        +


test14_BFS_Hanoi_03 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 314, in test14_BFS_Hanoi_03
    [23, ((1, 4, 5), (), (2, 3))],
  File "C:\Users\kille\academics\automatedgradingapp\app\testing\uninformed-solvers\gradingtests.py", line 55, in runPlayXSteps
    self.assertEqual(results[index], expected)
AssertionError: Tuples differ: ((3, 4, 5), (), (1, 2)) != ((3, 4, 5), (1, 2), ())

First differing element 1:
()
(1, 2)

- ((3, 4, 5), (), (1, 2))
?             ----

+ ((3, 4, 5), (1, 2), ())
?                   +++ +


-------------------------------
TESTS SKIPPED 
-------------------------------
